### ISSUE TITLE
Releases for 2.9.0-1 and 2.9.1

### ISSUE THREAD

**Issue by rossabaker** [djspiewak/anti-xml#42]
We crossbuilders would love to see an anti-xml release for Scala 2.9.0-1 and Scala 2.9.1.  2.9.0 is binary compatible, but depending on a different crossbuild is messy and can lead to classpath duplicates as more crossbuilds are released.

Is 0.3 imminent?  If not, could we do a retro release of 0.2 for these Scala versions?  I'm willing to submit a patch for either case.

**Comment by djspiewak** [comment]
I agree 100%.  Usually, I'm blocked on Specs2 and ScalaCheck releasing the appropriate cross-builds.  I'll look into it tonight and bug the appropriate project leads if the cross-build isn't available.

I'm actually a bit surprised that I didn't push a cross-build to 2.9.0-1 for 0.2.  That seems like a pretty major oversight on my part.

**Comment by rossabaker** [comment]
Thanks, Daniel!  I've already done some of that homework for Scalatra:
- specs2_2.9.1 is at 1.6-SNAPSHOT, expected release this weekend.
- specs2_2.9.0-1 is at 1.5 and 1.6-SNAPSHOT.
- scalacheck_2.9.1 is currently unavailable.
- scalacheck_2.9.0-1 is at 1.9.

**Comment by djspiewak** [comment]
Wow, ScalaCheck is really annoying with their cross-builds!  1.9 is available for everything in 2.9.x…except 2.9.1.final.  Meanwhile, back at 2.8.1, scalacheck 1.9 isn't anywhere to be found, we only have 1.8.  Unfortunately, 1.8 doesn't exist for 2.9.0-1 and up.  Cross-builds can be really troublesome if they aren't properly maintained.

Nothing that can't be solved by matching on `crossScalaVersionString`, but still annoying.  Cross-builds for Anti-XML 0.2 now pushed for:
- 2.8.1
- 2.9.0
- 2.9.0-1
- 2.9.1

**Comment by rossabaker** [comment]
Thanks for the quick update!  

We need a community standard for cross building.  I posted a [rough draft](https://gist.github.com/1185548) last night.  I plan to polish it and then harangue people with it.  Boring stuff, but so is matching on crossScalaVersionString when we could be writing real code.
